client/src/component/EventDialog/CreateEventDialog.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import {
+  makeStyles,
+} from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import FormLabel from "@material-ui/core/FormLabel";
+import Radio from "@material-ui/core/Radio";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormControl from "@material-ui/core/FormControl";
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    textAlign: "center",
+    color: theme.palette.text.secondary,
+    "& > *": {
+      margin: 8,
+      width: "80ch",
+    },
+  },
+}));
+
+export default function MaxWidthDialog(props) {
+  const [state, setState] = useState({
+    user_id: "",
+    event_name: "",
+    type: localStorage.getItem("type"),
+    description: "",
+    duration: 15,
+  });
+
+  const [selectedValue, setSelectedValue] = useState("15");
+
+  const handleChange = (event) => {
+    setSelectedValue(event.target.value);
+  };
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+  const [fullWidth] = React.useState(true);
+  const [maxWidth] = React.useState("lg");
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const inputChange = (event) => {
+    const value = event.target.value;
+    setState({
+      ...state,
+      [event.target.name]: value,
+    });
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const createEvent = (event) => {
+    fetch(
+      "http://localhost:5000/meeting/5f69713107ad65349c8ad946/meeting/one-on-one/company/45/",
+      {
+        method: "GET",
+        headers: { "Content-type": "application/json" },
+      }
+    )
```

Comment:
Since we are creating a meeting, it's better practice to use a POST method.
Also can we use the body instead of the URL for this? So many parameters makes it more confusing to know which is which.